src/libultra/os/exceptasm.s

```diff
@@ -308,7 +308,7 @@ cart:
     // Load cart callback set by __osSetHWIntrRoutine
     lui     $t1, %hi(__osHwIntTable)
     addiu   $t1, %lo(__osHwIntTable)
-    lw      $t2, (OS_INTR_CART*HWINT_SIZE+HWINT_CALLBACK)($t1)
+    lw      $t2, (OS_INTR_CART*HWINT_SIZE+HWINT_HANDLER)($t1)
```

Comment:
since `HWINT_CALLBACK` -> `HWINT_HANDLER` , should references to "callback" in sentences be changed too?